edx_name_affirmation/toggles.py

```diff
@@ -0,0 +1,19 @@
+"""
+Toggles for edx-name-affirmation app
+"""
+
+from edx_toggles.toggles import WaffleFlag
+
+# .. toggle_name: name_affirmation.verified_name
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Enable the verified name feature
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-06-25
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: MST-801
+VERIFIED_NAME_FLAG = WaffleFlag('name_affirmation.verified_name', __name__)
```

Comment:
Please make this a coursewaffleoverride type

Reply:
To follow up on this, were you able to use the `CourseWaffleFlag`? I assume not, because it's a custom flag defined in the LMS that you don't have access to from this repository. Where did you land on Simon's feedback above?

Reply:
Thank you for the update. That's useful.